src/main/java/com/company/managementservice/service/DepartmentService.java

```diff
@@ -0,0 +1,56 @@
+package com.company.managementservice.service;
+
+import com.company.managementservice.exception.NotFoundException;
+import com.company.managementservice.model.dto.DepartmentDto;
+import com.company.managementservice.model.dto.OrganisationDto;
+import com.company.managementservice.model.entity.Department;
+import com.company.managementservice.repo.DepartmentRepo;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class DepartmentService {
+
+    @Autowired
+    private DepartmentRepo departmentRepo;
+
+    private ModelMapper modelMapper = new ModelMapper();
```

Comment:
Create mapper package and mapper class. You can check in personalisation-service